modules/RTC/TraceablePeerConnection.js

```diff
@@ -1155,6 +1155,33 @@ const enforceSendRecv = function(localDescription) {
     return localDescription;
 };
 
+/**
+ * Neither Safari nor Chromium adds rtcp-mux to an application channel
+ * The function makes sure that it is set to pass sdp-interop validation
+ * @param {Object} localDescription the SDP object as defined by WebRTC.
+ */
+const injectRtcpMux = function(localDescription) {
```

Comment:
i think this would be better in `SdpTransformUtil.js`

Reply:
also, did this code already exist somewhere?  if not, how were things working?  electron uses chromium, so i would think it would've had this issue.  or is this only in the unified case? (in which case, i guess firefox does this correctly on its own?)

Reply:
Things are working because Chromium uses plan-b for now and it doesn't need sdp-interloop conversion, it's sdp-interloop that doesn't validate an SDP. There was [an issue for Firefox about the data channel and rtcp-mux](https://github.com/jitsi/sdp-interop/issues/13 ) on sdp-interop's GitHub, Firefox seems not to set rtcp-mux in some cases as well.

I anticipate that sdp-interloop is going to endure a lot of modifications as the question inevitably comes to Chrome's unified-plan, they are to drop plan-b next year or may be bit later.

Doesn't matter anyway. This piece of code will be removed because we decided to make Safari work in plan-b mode first, I'll update the diff soon.